config/application.rb

```diff
@@ -58,9 +58,15 @@ class Application < Rails::Application
     # we do here instead of using initializers because of a Rails 5.1 vs
     # MySQL bug where `rake db:reset` causes ActiveRecord to be loaded
     # before initializers and causes configuration not to be respected.
-    config.load_defaults 6.0
+    config.load_defaults 6.1
```

Comment:
@jlledom I applied the new defaults.

In the end after reading [this article](https://lilyreile.medium.com/rails-6-1-new-framework-defaults-what-they-do-and-how-to-safely-uncomment-them-c546b70f0c5e) again I got convinced that they are safe to apply:

- `cookies_same_site_protection`: Chrome, Firefox and Edge already apply `SameSite=lax` anyway, so the behavior will not change (and even if it did - it's a good thing, because none is insecure)
- `urlsafe_csrf_tokens` the old tokens are still gonna be decoded, and the newly generated work pretty much the same way.
So, it leaves us only with overriding `has_many_inversing`, which is already here.

So, I removed the "new defaults" file.